app/views/admin/teachers/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2><%= action_name == 'new' ? 'New' : 'Edit' %> <%= t('.teachers_title') %></h2>
+<br>
+<div class="container">
+  <%= simple_form_for [:admin, @teacher], defaults: { input_html: { class: 'form-control' } } do |f| %>
+    <%= f.input :subject_id, collection: Subject.all, label_method: :name, value_method: :id, include_blank: false %>
```

Comment:
define `@subjects = Subject.all` in controller

Reply:
I put it in controller

Reply:
I did differently. I made association in simple form and it works